packages/fx-core/src/component/driver/add/utility/constants.ts

```diff
@@ -5,7 +5,7 @@ export class Constants {
   static readonly ActionName = "spfx/add";
 
   static readonly LOCAL_CONTENT_URL =
-    "https://{teamSiteDomain}/_layouts/15/TeamsLogon.aspx?SPFX=true&dest={teamSitePath}/_layouts/15/TeamsWorkBench.aspx%3FcomponentId=%s%26teams%26personal%26forceLocale={locale}%26loadSPFX%3Dtrue%26debugManifestsFile%3Dhttps%3A%2F%2Flocalhost%3A4321%2Ftemp%2Fmanifests.js";
+    "https://{teamSiteDomain}/_layouts/15/TeamsLogon.aspx?SPFX=true&dest=/_layouts/15/teamshostedapp.aspx%3Fteams%26personal%26componentId=%s%26forceLocale={locale}%26loadSPFX%3Dtrue%26debugManifestsFile%3Dhttps%3A%2F%2Flocalhost%3A4321%2Ftemp%2Fmanifests.js";
```

Comment:
Why use teamshostedapp for local debug?

Reply:
just copy from as what this web page mentioned. https://learn.microsoft.com/en-us/sharepoint/dev/spfx/deployment-spfx-teams-solutions#dynamically-reference-the-underlying-sharepoint-site-urls